nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -176,7 +176,13 @@ define([
             fileTypePanel.updateState(getFileTypeState());
             model.setItem('state.selectedFileType', fileType);
             // stop and start the widgets.
-            return stopInputWidgets().then(startInputWidgets());
+            return stopInputWidgets()
+                .then(() => startInputWidgets())
+                .then(() => {
+                    cellBus.emit('toggled-active-filetype', {
```

Comment:
Added this to alert whatever's listening (mainly the test suite right now) that changes have been made and should be checked.